app/util/Youtube.scala

```diff
@@ -151,28 +149,26 @@ case class YouTubeVideoUpdateApi(config: YouTubeConfig) extends YouTubeBuilder {
     }
   }
 
-  private def setStatusToPrivate(asset: Asset, atomId: String): Unit = {
-    if (asset.platform == Youtube) {
-      YouTubeVideoInfoApi(config).getVideo(asset.id, "snippet,status") match {
-        case Some(video) => {
-          protectAgainstMistakesInDev(video)
+  def setStatusToPrivate(id: String, atomId: String): Unit = {
```

Comment:
Yeah, it will then make the requests to YouTube in parallel which is nice. Did you have any other reason for wanting it?